tests/security/selinux/sestatus.pm

```diff
@@ -17,7 +17,10 @@ use utils;
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
-    $self->set_sestatus("permissive", "minimum");
+    $self->set_sestatus('permissive', 'minimum');
+
+    # Check SELinux status: 'selinuxenabled' exits with status 0 if SELinux is enabled and 1 if it is not enabled
```

Comment:
could you please and some **","** in this sentence as it looks like 2 **if** in this comment.

Reply:
I just copied the sentence from man page. It is OK.